src/Confluent.Kafka/IAdminClient.cs

```diff
@@ -324,6 +324,26 @@ public interface IAdminClient : IClient
         /// </returns>
         Task<List<DeleteAclsResult>> DeleteAclsAsync(IEnumerable<AclBindingFilter> aclBindingFilters, DeleteAclsOptions options = null);
 
+        /// <summary>
+        ///    Delete committed offsets for a set of partitions in a conusmer
+        ///    group. This will succeed at the partition level only if the group
+        ///    is not actively subscribed to the corresponding topic.
+        /// </summary>
+        /// <param name="group">
+        ///     Consumer group id
+        /// </param>
+        /// <param name="partitions">
+        ///     Partitions to delete committed offsets for.
+        ///     Only the topic and partition fields are used.
+        /// </param>
+        /// <param name="options">
+        ///     The options to use when deleting the committed offset.
+        /// </param>
+        /// <returns>
+        ///     A Task returning a List of <see cref="Confluent.Kafka.Admin.DeleteAclsResult"/>.
+        /// </returns>
+        Task<List<DeleteConsumerGroupOffsetsResult>> DeleteConsumerGroupOffsetsAsync(string group, IEnumerable<TopicPartitionOffset> partitions, DeleteConsumerGroupOffsetsOptions options = null);
```

Comment:
Pass in `IEnumerable<TopicPartition>`, not `IEnumerable<TopicPartitionOffset>` because offset isn't relevant and having the user specify it is misleading.

Reply:
Makes sense, uppdated